themes/default/layouts/page/pricing.html

```diff
@@ -638,7 +638,7 @@ <h2 class="text-center">Compare editions</h2>
                         <span>1</span>
                     </div>
                     <div class="compare-plans-table-row-item">
-                        <span>Unlimited</span>
+                        <span>Custom</span>
```

Comment:
I think the original text here is more what was and is intended for the product as we actually offer it today - I'm not sure I'd change this yet until we've gotten a bit clearer on whether the cases that might motivate a change are actually intentional or not.